src/Smarkio/Supplier/Lead.php

```diff
@@ -585,6 +585,16 @@ public function setSmkSubCategory($smk_subcategory)
         $this->leadFields['smk_subcategory'] = $smk_subcategory;
     }
 
+    public function getSmkCreateNew()
+    {
+        return isset($this->leadFields['smk_create_new']) ? $this->leadFields['smk_create_new'] : null;
+    }
+
+    public function setSmkCreateNew($smk_create_new)
+    {
+        $this->leadFields['smk_create_new'] = $smk_create_new;
+    }
+
```

Comment:
I would do it differently. The purpose of the accelerator is to help people integrating leads - they don't need to know or care what is the format of the parameter to force the lead creation. I suggest changing this by:

```
public function isForceNewLeadCreation()
{
    return isset($this->leadFields['smk_create_new']) && $this->leadFields['smk_create_new'] === "1";
}

public function setForceNewLeadCreation($force=true)
{
	if( $forceNewLeadCreation )
	{
    	$this->leadFields['smk_create_new'] = "1";
    }
    else
    {
    	unset($this->leadFields['smk_create_new']);
    }
}
```

> Not tested this code. ;-)